src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionUtilityToolFilter.java

```diff
@@ -121,22 +122,32 @@ private boolean isEmpty(String value) {
 	 * 
 	 */
 	private boolean checkForAuthorization(HttpServletRequest request) {
-		M_log.debug("checkLdapForAuthorization(): called");
+		M_log.debug("checkLdapForAuthorization(): called");		
 		String remoteUser = request.getRemoteUser();
 		String testUser = request.getParameter(TEST_USER);
+		String ltiTestUser = request.getParameter(CANVAS_ID);
 		boolean isAuthorized = false;
 		String user=null;
 
 		String testUserInSession = (String)request.getSession().getAttribute(TEST_USER);
 		String sessionId = request.getSession().getId();
 
-		if ( isTestUrlEnabled && testUser != null ) { 
+		if( isTestUrlEnabled && ltiTestUser != null ) {
```

Comment:
Why not just check isTestUrlEnabled once, since it needs to be true for all these tests?